src/parser/bgp/attributes/mod.rs

```diff
@@ -189,30 +179,28 @@ impl AttributeParser {
                 AttrType::DEVELOPMENT => {
                     let mut value = vec![];
                     for _i in 0..attr_length {
-                        value.push(attr_data.get_u8());
+                        value.push(attr_data.read_u8()?);
                     }
                     Ok(AttributeValue::Development(value))
                 }
                 AttrType::ONLY_TO_CUSTOMER => parse_only_to_customer(attr_data),
-                _ => Err(ParserError::Unsupported(format!(
-                    "unsupported attribute type: {:?}",
-                    attr_type
-                ))),
+                // TODO: Should it be treated as a raw attribute instead?
+                _ => Err(ParserError::UnsupportedAttributeType(attr_type)),
```

Comment:
Unsupported is fine in my opinion. We can add raw attribute support later.

Reply:
I think you already implemented raw attribute support. It seems this line is only reachable it it finds an attribute type it recognizes, but does not get parsed.

Reply:
How about?
```suggestion
                _ => Ok(AttributeValue::Unknown(AttrRaw{ attr_type, bytes: Vec::from(attr_data) }))
```

We could probably introduce a new `AttributeValue` type `Unsupported(AttrRaw)`, but I feel that's a bit unnecessary.